doc/reorder.awk

```diff
@@ -0,0 +1,115 @@
+# This AWK script extracts typescripts comments from lsp-messages.ads
+# and reorders them in an order that corresponds to LSP specification.
+# Use next command to run the script:
```

Comment:
* Use the following command to run the script:

Reply:
:heavy_check_mark: done!